src/main/java/org/prebid/server/auction/BidsAdjuster.java

```diff
@@ -95,8 +95,9 @@ private AuctionParticipation validBidderResponse(AuctionParticipation auctionPar
 
         final List<String> requestCurrencies = bidRequest.getCur();
         if (requestCurrencies.size() > 1) {
-            errors.add(BidderError.badInput("Cur parameter contains more than one currency. %s will be used"
-                    .formatted(requestCurrencies.getFirst())));
+            warnings.add(BidderError.badInput(
+                    "a single currency (%s) has been chosen for the request. ".formatted(requestCurrencies.getFirst())
+                            + "ORTB 2.6 requires that all responses are in the same currency."));
```

Comment:
Please fix the code by using either concatenation (+) or the .formatted() method consistently for string formatting, rather than mixing both.